[debug] [2019-01-27T12:48:00.224Z] ----------------------------------------------------------------------
[debug] [2019-01-27T12:48:00.226Z] Command:       /usr/bin/node /usr/local/bin/firebase serve
[debug] [2019-01-27T12:48:00.227Z] CLI Version:   6.1.2
[debug] [2019-01-27T12:48:00.227Z] Platform:      linux
[debug] [2019-01-27T12:48:00.227Z] Node Version:  v8.15.0
[debug] [2019-01-27T12:48:00.227Z] Time:          Sun Jan 27 2019 18:18:00 GMT+0530 (IST)
[debug] [2019-01-27T12:48:00.227Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-01-27T12:48:00.235Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-01-27T12:48:00.235Z] > authorizing via signed-in user
[debug] [2019-01-27T12:48:00.236Z] [iam] checking project todo-anotheryet for permissions ["firebase.projects.get"]
[debug] [2019-01-27T12:48:00.238Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/todo-anotheryet:testIamPermissions  

[debug] [2019-01-27T12:48:01.703Z] <<< HTTP RESPONSE 200
[debug] [2019-01-27T12:48:01.709Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/todo-anotheryet  
 
[debug] [2019-01-27T12:48:03.184Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/home/suresh/workspace/anotheryet/todo'...
[info] 
[debug] [2019-01-27T12:48:03.713Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
